packages/http/README.md

```diff
@@ -10,6 +10,10 @@ In essence it's an HTTP client (similar to axios - in fact, we use axios to make
 
 The goal of this document is to provide you with some basic code examples to get you started and to cover some of the advanced scenarios.
 
+## Important
+
+If you're a regular user and not a PRO, you might to want to use the [User facing client](../../docs/guides/client.md) which
```

Comment:
what is the difference between regular and pro? also message is chopped

Reply:
The "old", or be precise, the internal client, offers a single method and the idea is that this will have breaking changes over time, that's why we want to suggest *NOT* to use it.

Request Maker should migrated from that.

Reply:
The internal client is just that, internal. It's not documented and maybe going away or changing drastically over time. The HTTP Client is what people should be using, so let's just talk about the HTTP Client?

Users are either:

- Using Prism CLI
- Using Prism HTTP Client
- Using Prism HTTP Server? 

We didn't document how to use the HTTP server yet. We've only ever talked about CLI so far as I know.

Reply:
So you're fundamentally suggesting to remove this page, right?

Reply:
@XVincentX, can we finish the sentence? It suddenly ends with `which`